angular-portfolio/src/app/mock-research.ts

```diff
@@ -0,0 +1,16 @@
+import { Res } from './resTemplate';
+
+export const RESES: Res[] = [
+  { title: 'Massachusetts General Hospital', date: 'November 2016 - Aug 2018', outcome: 'Solved a \
```

Comment:
For multi-line strings, use backtics instead of "\". See rules 5.6.2 and 5.6.3: https://google.github.io/styleguide/jsguide.html#features-strings-template-strings

Reply:
Sorry, I was trying to show \\ but accidentally escaped the character.